routers/repo/projects.go

```diff
@@ -286,10 +286,32 @@ func ViewProject(ctx *context.Context) {
 	allBoards := models.ProjectBoardList{uncategorizedBoard}
 	allBoards = append(allBoards, boards...)
 
-	if ctx.Data["Issues"], err = allBoards.LoadIssues(); err != nil {
+	issueList, err := allBoards.LoadIssues()
+	if err != nil {
 		ctx.ServerError("LoadIssuesOfBoards", err)
 		return
 	}
+	ctx.Data["Issues"] = issueList
+
+	linkedPrsMap := make(map[int64][]*models.Issue)
+	for _, issue := range issueList {
+		var referencedIds []int64
+		for _, comment := range issue.Comments {
```

Comment:
are you sure that the comments are actually loaded?

Reply:
Comments are fetched in `func (b *ProjectBoard) LoadIssues()`: https://github.com/go-gitea/gitea/pull/14183/files#diff-14e6aa7df6f43549ba13b9a18fe6ed6ae69ca35f7003ee3ad2ed1e24a3a763bfR208